specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ManagedEnvironmentsMaintenanceConfigurations.json

```diff
@@ -0,0 +1,361 @@
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "MaintenanceConfigurations"
+        ],
+        "operationId": "MaintenanceConfigurations_Get",
+        "x-ms-examples": {
+          "ManagedEnvironmentMaintenanceConfigurationsGet": {
+            "$ref": "./examples/ManagedEnvironment_MaintenanceConfigurations_Get.json"
+          }
+        },
+        "description": "Gets the maintenance configuration of a ManagedEnvironment .",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "description": "The name of the Managed Environment.",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          },
+          {
+            "name": "configName",
+            "description": "The name of the maintenance configuration.",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response of get maintenance configurations.",
+            "schema": {
+              "$ref": "#/definitions/MaintenanceConfigurationResource"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ScheduledEntry": {
+      "type": "object",
+      "description": "Maintenance schedule entry for a managed environment.",
+      "properties": {
+        "weekDay": {
+          "type": "string",
+          "description": "Day of the week when a managed environment can be patched.",
+          "enum": [
+            "Monday",
+            "Tuesday",
+            "Wednesday",
+            "Thursday",
+            "Friday",
+            "Saturday",
+            "Sunday",
+            "Everyday",
+            "Weekend"
+          ],
+          "x-ms-enum": {
+            "name": "WeekDay",
+            "modelAsString": false
+          }
+        },
+        "startHourUtc": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Start hour after which managed environment maintenance can start from 0 to 23 hour."
+        },
+        "durationHours": {
+          "type": "integer",
+          "format": "int32",
```

Comment:
>int32 [](http://example.com/codeflow?start=21&length=5)

you can also add a minimum and maximum for it

Reply:
same as above